### PR TITLE
Created NavigationMenu and added Actions with cart test

### PR DESCRIPTION
**Pull request by asukhodolova**

### CODE REVIEW

File: src/main/java/com/solvd/mydemoapp/mobile/pages/android/CartScreen.java [hunk 1]
```diff
@@ -64,8 +75,57 @@ public List<String> getAddedProductNames() {
...
+
     @Override
     public boolean isOpened() {
         return proceedToCheckoutButton.isElementPresent();
     }
+
+    @Override
+    public INavigationMenu getNavigation() {
+        return new NavigationMenu(getDriver());
+    }
```
Comment (ankochergin) [inline src/main/java/com/solvd/mydemoapp/mobile/pages/android/CartScreen.java:121]:
```suggestion
    @Override
    public NavigationMenuBase getNavigation() {
        return initPage(NavigationMenuBase.class);
    }
```

File: src/main/java/com/solvd/mydemoapp/mobile/pages/ios/NavigationMenu.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    @ExtendedFindBy(iosClassChain = "**/XCUIElementTypeOther[-1]/XCUIElementTypeStaticText")
+    private ExtendedWebElement cartAmountLabel;
+
+    public NavigationMenu(WebDriver driver) {
+        super(driver);
+    }
+
+    public CartScreenBase openCart() {
+        cartButton.click();
+        return initPage(getDriver(), CartScreenBase.class);
```
Comment (ankochergin) [inline src/main/java/com/solvd/mydemoapp/mobile/pages/ios/NavigationMenu.java:25]:
```suggestion
        return initPage(CartScreenBase.class);
```

Comment (ankochergin) [reply src/main/java/com/solvd/mydemoapp/mobile/pages/ios/NavigationMenu.java:25]: can be simplified, you can remove getDriver() in all `initPage`s

File: src/main/java/com/solvd/mydemoapp/mobile/pages/android/CartScreen.java (hunk 1 above)
Comment (asukhodolova) [reply src/main/java/com/solvd/mydemoapp/mobile/pages/android/CartScreen.java]: Fixed

File: src/main/java/com/solvd/mydemoapp/mobile/pages/ios/NavigationMenu.java (hunk 2 above)
Comment (asukhodolova) [reply src/main/java/com/solvd/mydemoapp/mobile/pages/ios/NavigationMenu.java]: Fixed

**Pull request merged**